quipucords/tests/constants.py

```diff
@@ -19,7 +19,7 @@
 QPC_COMMIT = "test-commit"
 QPC_SERVER_PASSWORD = "test-password"
 QPC_SERVER_USERNAME = "test-username"
-SCAN_TARGET_USERNAME = "non-root-user"
+READINESS_TIMEOUT_SECONDS = 60
```

Comment:
Why did you have to change the order?

Reply:
alfanumeric order lol. I'll change the commits to not have this juggling around